apps/iis.go

```diff
@@ -95,6 +114,7 @@ func (r MetricsReceiverIis) Pipelines() []otel.Pipeline {
 			otel.NormalizeSums(),
 		},
 	}}
+
```

Comment:
Any particular reason for this extra blank line?

Reply:
Removed